### ISSUE TITLE
Black output image

### ISSUE THREAD

**Issue by mlbrothers** [Sun-ZhenXing/Real-ESRGAN-android#2]
As title, output image is totally black. I think somewhere * 255 needed in output but I am not familiar with cpp much. Can someone guide?

**Comment by Sun-ZhenXing** [comment]
If it appears to be all black, you should first test if it is a problem with Real-ESRGAN NCNN, the test is to download its pre-compile tool and execute it directly using the command line.

This may be a known problem with Real-ESRGAN, but normal images are fine with me. I did not change the C++ code, but just wrapped and compiled the interface with the help of the NDK.

**Comment by mlbrothers** [comment]
Yeah, it looks like issue with NCNN model. I will update if solved.

**Comment by mlbrothers** [comment]
When I used models from your repo with [Real-ESRGAN-ncnn-vulkan](https://github.com/xinntao/Real-ESRGAN-ncnn-vulkan) exe, it worked fine. Any idea what could be an issue here?

**Comment by Sun-ZhenXing** [comment]
Can you provide a sample image of the error?

In fact, my shared library is compiled on a native ARM device, and debugging is quite difficult for me.

I think you can convert the images to different formats for testing.

If given the opportunity, I would like to compile the entire project using native NDK. Unfortunately, currently this project is only a toy, and I do not have the time and energy to continue maintaining it.

There are also some well maintained projects here, you can take a look at the code they provide: https://github.com/tumuyan/RealSR-NCNN-Android.
